### ISSUE TITLE
Imported product sometimes has blank handle

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#4011]
## Issue Description
Sometimes when the Basic Reaction Product is imported on startup it gets imported without a handle which causes the grid to crash with this error:
```
TypeError: Expected "handle" to match "[^\/]+?", but got ""
```

## Steps to Reproduce
1. Start the app with `reaction-devtools` installed
1. Do "reset data" and load small data-set (everything will be fine)
1. Stop the app and start it again so that "Basic Reaction Product" gets imported
1. Refresh
1. Observe that the grid does not load
1. If you look in Mongo observe that the handle for the "Basic Reaction Product" is blank

### Versions
```
Node: 8.9.4
NPM: 5.6.0
Meteor Node: 8.9.4
Meteor NPM: 5.6.0
Reaction CLI: 0.29.0
Docker: 17.12.0-ce
```

**Comment by prinzdezibel** [comment]
Happens if you delete the products in db manually and restart the server. Then the default product is inserted , but somehow the handle got lost.

**Comment by zenweasel** [comment]
When creating products manually I am sometimes getting a blank title which is resulting in a blank handle

**Comment by zenweasel** [comment]
Fixing this since this happens almost constantly when working on devtools

**Comment by zenweasel** [comment]
Closed via #4049
